Gamecube/libgui/Gui.cpp

```diff
@@ -128,7 +128,11 @@ void Gui::draw()
 					*(volatile unsigned int*)0xCC003024 = 0;  //reboot
 			  }
 #else
-				rld();
+				if(*(volatile unsigned int*)0x80001804 == 0x53545542 &&

```

Comment:
I'm all for code that works, but can you add a comment here about these "magic numbers", or add some defines? I'm not familiar with what this is memory space that's dereferenced and the value it's expecting it to be. I'd rather not make this code more messy than it already is.